### PR TITLE
Optimize listen function

### PR DESCRIPTION
**Pull request by Aerion**
- Use grep first to receive only the wanted lines
- Flush the pending events to call output only once

With multiple occurrences of pulseaudio-control started, I noticed
that on quick changes (i.e. volume or iterating through the sinks),
the cpu usage increased a lot for the action.

pactl is quite noisy, and a single event can generate multiple
matching lines.

This commit reduces the calls to output to one per 100ms.

### CODE REVIEW

File: pulseaudio-control.bash [hunk 1]
```diff
@@ -283,26 +283,18 @@ function showOSD() {
 
 
 function listen() {
-    local firstRun=0
+    # If this is the first time start by printing the current
```
Comment (marioortizmanero) [inline pulseaudio-control.bash]: It seems like the line breaks in your comments are of inconsistent length. Could you fix them?

**Review by marioortizmanero — COMMENTED** [review]
Love it! Lots of infrequently-used flags. Great idea, thanks a bunch for the PR.

File: pulseaudio-control.bash (hunk 1 above)
Comment (Aerion) [reply pulseaudio-control.bash]: Sure thing! The comments now wrap to 80 col, let me know if you had something else in mind

**Comment by marioortizmanero** [comment]
Awesome. I've roughly measured the performant effect and you can actually see it. When generating lots of events (mainly increasing or decreasing the volume rapidly), htop reports 8-9% CPU load with the previous implementation. Now it's always under 2%.

Props for the idea! Merging once CI runs, thanks again.

**Pull request merged**